news.rst

```diff
@@ -12,6 +12,19 @@ for which `Damian Avila <https://github.com/damianavila>`_ was a technical
 reviewer. We thank Packt Publishing for donating a portion of the proceeds
 from this book to support IPython's development.
 
+IPython 4.0
+-----------
+
+IPython 4.0 was release on August 11, 2015. This release correspond to the
+separation of IPython (as of 3.x) into multiple components, most of which are
+now under the Jupyter organisation. See `the jupyter blog post
+<https://blog.jupyter.org/2015/04/15/the-big-split/`_ for further explanations.
```

Comment:
explanation

(explanations is valid, but sounds a bit odd - it's usually used as a mass noun, like 'flour')